test/no-abstract-functions.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @fileoverview Test for no-abstract-functions rule
+ * @author Nicolas Feignon <nfeignon@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require ('solium');
+
+var userConfig = {
+    rules: {
+        "security/no-abstract-functions": "error"
+    }
+};
+
+describe ('[RULE] no-abstract-functions', function () {
```

Comment:
Add `: Rejections` at the end of description. There is no need for an `Acceptances` test section for this rule.

Please add a couple more rejection tests with different kind of function signatures, like

```
function foo(uint x, string y);
function foo(uint x) returns (uint);
function foo() payable public;
function foo(uint x, string y) MyOwnModifier;

function();         // fallback functions can also be abstract
```